00_Base/src/interfaces/modules/AbstractModule.ts

```diff
@@ -144,6 +155,17 @@ export abstract class AbstractModule implements IModule {
         }
     }
 
+    async handleMessageApiCallback(message: IMessage<OcppResponse>): Promise<void> {
+        const url: string | null = await this._cache.getAndRemove(message.context.correlationId, this.CALLBACK_URL_CACHE_PREFIX + message.context.stationId);
+        if (url) {
+            try {
+                await axios.post(url, message.payload);
```

Comment:
@thanaParis I would recommend to use `NodeJS` fetch instead of `Axios` as `Axios` adds another dependency and is not necessary to make a simple post request.